.github/zizmor.yml

```diff
@@ -9,13 +9,18 @@ rules:
   cache-poisoning:
     ignore:
       - build-docker.yml
-      - publish-playground.yml
-      - ty-ecosystem-analyzer.yaml
-      - ty-ecosystem-report.yaml
```

Comment:
I've removed these, since we now use inline ignore comments to ignore the exact acceptable cache risks rather than blanket ignoring all cache poisoning vectors 🙂 

(I did a review pass on each of these, and the primary risk in all cases is that a poisoned cache could influence one of the various HTML artifacts pushed to CloudFlare Pages. I believe that's an acceptable risk in practice!)